cirq-core/cirq/ops/pauli_string.py

```diff
@@ -1237,6 +1237,7 @@ def _imul_helper(self, other: 'cirq.PAULI_STRING_LIKE', sign: int):
         Returns:
             self on success, NotImplemented given an unknown type of value.
         """
+        from cirq.ops.linear_combinations import PauliSum
```

Comment:
Nit: can we move this to a module level import at the top ? (even if it means wrapping in LazyLoader)

Reply:
Totally forgot about `LazyLoader`. Thank you. Done.